package.json

```diff
@@ -15,10 +15,10 @@
     "test": "npm run build && npm run test:unit",
     "test:all": "npm run test:unit && npm run test:integration",
     "test:all:parallel": "npm run test:unit && npm run test:integration:parallel",
-    "test:integration": "lerna run test:integration --stream",
+    "test:integration": "lerna run test:integration --stream --concurrency 1",
     "test:integration:parallel": "lerna run test:integration --stream --parallel",
     "test:integration:windows": "lerna run test:integration --stream --ignore web-component-tester --ignore @polymer/esm-amd-loader --parallel",
-    "test:unit": "lerna run test:unit --stream",
+    "test:unit": "lerna run test:unit --stream --concurrency 1",
```

Comment:
how does this affect our testing performance?

Reply:
Unit test time impact is negligible and it simplifies output.  If higher concurrency we got a real bump in test time on travis without impacting reliability I'd bump it up.

Reply:
(by negligible I mean not consistently more-than it was.)